### PR TITLE
GSA-6547 tutorial videos

### PR DESCRIPTION
**Pull request by kpne**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,15 @@ Head over to our GSA documentation site for:
...
 
-Tutorials and more example files to come shortly!
+## Tutorials
+[![Tutorials](https://img.shields.io/badge/Tutorials-Start%20today-red?logo=youtube&style=social)](https://www.youtube.com/playlist?list=PLm0cokV2wawoKLWO7cp8FBlhGkBsbEYsh)
+
+Started learning GsaGH by following the tutorials:
+- [Installing GsaGH](https://docs.oasys-software.com/structural/gsa/tutorials/gsagh-installing-grasshopper-plugin.html)
+- [Initiation and working with existing models](https://docs.oasys-software.com/structural/gsa/tutorials/gsagh-initiation-existing-models.html)
+- [Creating profiles](https://docs.oasys-software.com/structural/gsa/tutorials/gsagh-creating-profiles.html)
+- [Making a basic 1D member model](https://docs.oasys-software.com/structural/gsa/tutorials/gsagh-basic-1D-member-model.html)
```
Comment (tlmnrnhrdt) [inline README.md:26]: Creating instead of making?

File: README.md [hunk 2]
```diff
@@ -16,7 +16,15 @@ Head over to our GSA documentation site for:
 - [Parameters overview](https://docs.oasys-software.com/structural/gsa/explanations/gsagh-parameters.html?source=GsaGhGithub)
 - [Components overview](https://docs.oasys-software.com/structural/gsa/explanations/gsagh-components.html?source=GsaGhGithub)
 
-Tutorials and more example files to come shortly!
+## Tutorials
+[![Tutorials](https://img.shields.io/badge/Tutorials-Start%20today-red?logo=youtube&style=social)](https://www.youtube.com/playlist?list=PLm0cokV2wawoKLWO7cp8FBlhGkBsbEYsh)
+
+Started learning GsaGH by following the tutorials:
+- [Installing GsaGH](https://docs.oasys-software.com/structural/gsa/tutorials/gsagh-installing-grasshopper-plugin.html)
+- [Initiation and working with existing models](https://docs.oasys-software.com/structural/gsa/tutorials/gsagh-initiation-existing-models.html)
```
Comment (tlmnrnhrdt) [inline README.md:24]: Initialisation?

File: README.md [hunk 3]
```diff
@@ -16,7 +16,15 @@ Head over to our GSA documentation site for:
 - [Parameters overview](https://docs.oasys-software.com/structural/gsa/explanations/gsagh-parameters.html?source=GsaGhGithub)
```
Comment (tlmnrnhrdt) [inline README.md:16]: singular? `Parameter overview`

File: README.md [hunk 4]
```diff
@@ -16,7 +16,15 @@ Head over to our GSA documentation site for:
...
 - [Components overview](https://docs.oasys-software.com/structural/gsa/explanations/gsagh-components.html?source=GsaGhGithub)
```
Comment (tlmnrnhrdt) [inline README.md:17]: here as well?

**Review by tlmnrnhrdt — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (kpne) [reply README.md:24]: just copied the name of the page

**Pull request merged**